crates/biome_js_analyze/src/lint/style/use_naming_convention.rs

```diff
@@ -613,6 +671,16 @@ declare_lint_rule! {
     /// }
     /// ```
     ///
+    /// Hers some examples:
+    /// 
+    /// - A private class property named `_` is reported by the rule because it contains a single character.
+    ///   According to the second convention, the name should contain at least two characters.
+    /// - A variable named `a_variable` is reported by the rule because the name doesn't respect the default convention that forbid variable names in [`snake_case`].
```

Comment:
```suggestion
    /// - A variable `a_variable` is reported by the rule because it doesn't respect the default convention that forbid variable in [`snake_case`].
```

You often use "name" but it's redundant because we're documenting a rule that controls "names"